wordpress-mobile/WordPress-iOS
# Add ar-SA and he locales to screenshot generation

This PR adds the `ar-SA` (Arabic) and `he` (Hebrew) locales to screenshot generation. 

Ref: https://github.com/wordpress-mobile/WordPress-iOS/pull/16571#pullrequestreview-685552562

## Merge directions
- Merge after https://github.com/wordpress-mobile/WordPress-iOS/pull/16571

## Notes
- Flipping the text alignment for RtL languages needs to be handled in a separate PR. The splash image has text that's left-aligned for LtR languages, which needs to be flipped to be right-aligned for RtL languages. (See: the first image in the Jetpack screenshots section below.) I think this involves updating the [draw_text_attachment_to_canvas](https://github.com/wordpress-mobile/release-toolkit/blob/3417301e6104976d1413ce8282e84fd600738993/lib/fastlane/plugin/wpmreleasetoolkit/helper/promo_screenshots_helper.rb#L175-L199) method in Release Toolkit.

## Screenshots

**JP**
![iPhone XS Max-0-Splash](https://user-images.githubusercontent.com/6711616/122704942-af694480-d28f-11eb-8a73-1ba7a5ea66d5.png) | ![iPhone XS Max-1-MySite](https://user-images.githubusercontent.com/6711616/122704945-b09a7180-d28f-11eb-8915-861fdc6b79f4.png) | ![iPhone XS Max-2-ActivityLog](https://user-images.githubusercontent.com/6711616/122704949-b2643500-d28f-11eb-8a84-15a32d7380dd.png) | ![iPhone XS Max-3-JetpackScan](https://user-images.githubusercontent.com/6711616/122704953-b3956200-d28f-11eb-8b1c-f36836c8651d.png) | ![iPhone XS Max-4-JetpackBackup](https://user-images.githubusercontent.com/6711616/122704957-b55f2580-d28f-11eb-8130-483dcb996969.png) | ![iPhone XS Max-5-Stats](https://user-images.githubusercontent.com/6711616/122704960-b6905280-d28f-11eb-8325-fa39e3fe28a5.png)
-- | -- | -- | -- | -- | -- 

**WP**
![iPhone XS Max-0-Themes](https://user-images.githubusercontent.com/6711616/122704700-2b16c180-d28f-11eb-827c-e30f8cbba1c3.png) | ![iPhone XS Max-1-PostEditor](https://user-images.githubusercontent.com/6711616/122704719-34079300-d28f-11eb-8c49-1a2d5734215c.png) | ![iPhone XS Max-2-Reader](https://user-images.githubusercontent.com/6711616/122704727-3669ed00-d28f-11eb-87ca-9c233f852d0e.png) | ![iPhone XS Max-3-Stats](https://user-images.githubusercontent.com/6711616/122704735-3964dd80-d28f-11eb-9006-6daa795002aa.png) | ![iPhone XS Max-4-MySite](https://user-images.githubusercontent.com/6711616/122704736-3a960a80-d28f-11eb-97e3-1db029229d7b.png) | ![iPhone XS Max-5-Notifications](https://user-images.githubusercontent.com/6711616/122704741-3d90fb00-d28f-11eb-99db-180579510fa3.png) | ![iPhone XS Max-6-Media](https://user-images.githubusercontent.com/6711616/122704747-41248200-d28f-11eb-8b1c-5708bfa242f4.png)
-- | -- | -- | -- | -- | -- | --

## How to test 
1. Run `rake mocks`
2. Generate raw screenshots via `bundle exec fastlane jetpack_screenshots` (You can comment out all languages besides ar-sA and he)
3. ✅ Make sure raw screenshots are generated under `fastlane/jetpack_screenshots`


## Regression Notes
1. Potential unintended areas of impact
WP screenshot generation

2. What I did to test those areas of impact (or what existing automated tests I relied on)
I tested screenshot generation for both JP and WP.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
